src/main/resources/com/google/api/codegen/go/sample.snip

```diff
@@ -14,6 +14,8 @@
     //    If not already done, install the gcloud CLI from
     //    https://cloud.google.com/sdk/ and run
     //    `gcloud beta auth application-default login`
+    //    For more information, see:
```

Comment:
I would prefer adding a period after the second backtick so that we just have sentences with line breaks. Hanging indent doesn't seem right to me for this case.